version.txt

```diff
@@ -1 +1 @@
-7.62.1
+7.63.0-CItest
```

Comment:
Depending on when this PR is merged, and of we have manually published a docker image, we may want to change it.
No docker image for 7.63.0 has been published then leave it as is, the automation will publish.
If we have already published then, edit this to `7.63.0` to avoid an error the first night, as the git tag will already exist.